scripts/verify_tla_bridge_coverage.py

```
@@ -0,0 +1,204 @@
+
+def find_bridges(name: str, hits: list[BridgeHit]) -> list[tuple[str, str]]:
+    """Return [(fn_name, relpath)] for bridge proofs matching a TLA+ name."""
+    target = _norm(name)
+    results = []
+    for fn_name, relpath, tla_ref in hits:
+        norm_ref = _norm(tla_ref)
+        if norm_ref == target:
+            results.append((fn_name, relpath))
+        # Handle Spec::Action format and multi-action refs (e.g. "Dispatch + SpawnAndDispatch")
+        elif target in norm_ref:
+            results.append((fn_name, relpath))
```

Ruff (0.15.10)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)